src/main/kotlin/com/cosmotech/connector/adt/utils/JsonUtil.kt

```diff
@@ -52,13 +41,14 @@ class JsonUtil {
          * @return a Pair(Boolean,String) containing minimal extension information
          */
         @JvmStatic
-        fun isExtension(jsonModel: JsonObject):Pair<Boolean,String> {
-            return if( jsonModel.containsKey(DTDL_EXTENDS_KEY) )
-                Pair(true,jsonModel[DTDL_EXTENDS_KEY].toString())
+        fun isExtension(jsonModel: JsonObject):Pair<Boolean,List<String>> {
+            val result = jsonModel.array<String>(DTDL_EXTENDS_KEY)
```

Comment:
This line will throw an error if `extends` is a String, which remains a valid form.
Per [this page](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-models#model-inheritance) and https://github.com/Azure/opendigitaltwins-dtdl/issues/104#issuecomment-738207892, `extends` can be either a String or an Array.

These forms reportedly are all valid:

- `"extends": "dtmi:com:example:Base;1"`
- `"extends": [ "dtmi:com:example:Base;1" ]`
- `"extends": [ "dtmi:com:example:Base;1", "dtmi:com:example:AnotherInterface;1" ]`

So we need to support these cases then.

Please consider adding test cases in `JsonUtilsTest`.

Reply:
done